lib/stat_tracker.rb

```diff
@@ -1,7 +1,7 @@
 require 'csv'
-require_relative 'games'
-require_relative 'teams'
-require_relative 'game_teams'
```

Comment:
We renamed the files to singular nouns to match the class names. Going between games.rb for the Game Class, etc. was getting confusing. This helps to keep us organized for sure.